ansibleflow/run.py

```diff
@@ -18,9 +18,12 @@ def convert_var_filename_to_arg(filename):
     return ' -e @{0}'.format(os.path.abspath(filename))
 
 
-def build_ansible_command(playbook, target, environment):
+def build_ansible_command(playbook, target, environment, arguments):
     command = 'ansible-playbook'
 
+    if arguments.verbosity > 0:
+        command += ' -{0}'.format('v' * arguments.verbosity)
```

Comment:
Shouldn't this be a target configuration option instead of bringing this in via a cli argument?